src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataConstants.java

```diff
@@ -145,4 +148,13 @@ class HistoricalDataConstants {
 
   /** The IO server used for queue diagnostic tag(s). */
   public static final String QUEUE_DIAGNOSTIC_TAG_IO_SERVER = "MEM";
+
+  /** Start index of the actual ISO 8601 timestamp return by EBD. */
+  public static final int EBD_ISO8601_TIMESTAMP_START_INDEX = 1;
+
+  /** String index where milliseconds should be added to a timestamp that lacks milliseconds. */
+  public static final int EBD_ISO8601_TIMESTAMP_START_MILLIS = 20;
+
+  /** String to add to a timestamp that lacks milliseconds. */
+  public static final String EBD_ISO8601_TIMESTAMP_ADD_MILLIS_STR = ".000";
```

Comment:
This is more of a question than an issue, but was there an issue parsing without manually adding the `.000`?

Reply:
This is for parsing but also because most connectors ( this true for Azure - not sure about the others ) have ISO 6801 timestamps with millisecond resolution. I didn't want to introduce seconds and have the connectors add milliseconds for compatibility.   
Actually, I made a seconds timestamp and a milliseconds timestamps (and two getter methods) and then deleted the seconds because I thought it was unnecessary bloat.